frontend/src/components/AppointmentCard.tsx

```diff
@@ -0,0 +1,38 @@
+import './AppointmentCard.scss';
+import {useContext, useEffect, useState} from "react";
+import axios from "axios";
+import {AuthContext} from "../context/AuthProvider";
+
+
+export default function AppointmentCard() {
+
+    const [appointments, setAppointments] = useState([])
+
+    const {token} = useContext(AuthContext)
+
+    useEffect(() => {
+        axios.get("/api/appointments", token? {
```

Comment:
Imho this belongs inside the RequestService

Reply:
Yeah, this will be fixed as soon as i clean up the code